pkg/build/controller/strategy/custom.go

```diff
@@ -88,7 +88,8 @@ func (bs *CustomBuildStrategy) CreateBuildPod(build *buildv1.Build, additionalCA
 		serviceAccount = buildutil.BuilderServiceAccountName
 	}
 
-	privileged := true
+	securityContext := securityContextForBuild(strategy.Env)
+	secretsMode := int32(0600) // todo: change to corev1.SecretVolumeSourceDefaultMode (0644)?
```

Comment:
Do we want to do this for `Custom` strategy builds? Outside of OSBS this isn't really used (though this may be a vote in favor of enabling this capability).